destroy.js

```diff
@@ -1,13 +1,23 @@
 const core = require('@actions/core');
 const exec = require('@actions/exec');
+const util = require('util');
 const configure = require('./configure.js');
 
+const sleep = util.promisify(setTimeout);
+
 async function destroy() {
+  let name;
   try {
-    const name = core.getInput('name').replace(/\./g, '-');
+    name = core.getInput('name').replace(/\./g, '-');
     await exec.exec('gcloud container clusters delete --quiet', [name]);
-  } catch (e) {
-    core.setFailed(e.message);
+  } catch (e1) {
+    core.warning('error deleting cluster; attempting again in 5 minutes: ' + e1.message);
```

Comment:
Was 5 minutes picked arbitrarily? If so, maybe this should be set to a longer timeout so that it's more likely the cluster has finished its `create` step.

The reason for this change is because the step was cancelled during cluster creation, and therefore deletion immediately failed. However, cluster `create` can take longer than 5 minutes--or at least right around that general timeframe.

Maybe a timeout like 10 or 15 minutes would be safer?

Reply:
In my tests the cluster creation took about 4 minutes, so I set it to 5 to give it a little more margin. I still can set it to 10 to be more on the safer side :+1: